pkg/controller/bootstrap/bootstrap.go

```diff
@@ -163,13 +164,18 @@ func (b *Bootstrap) Run(destDir string) error {
 	if err != nil {
 		return err
 	}
+
+	klog.Infof("Bootstrap MCC base machine configs generation was successful")
```

Comment:
Nit: Maybe something like "Successfully generated MachineConfigs from templates"

Basically the "MCC" is a bit redundant since these logs should only show up in the collected MCC logs